msal/src/Microsoft.Identity.Client/Platforms/netstandard1.3/UIParent.cs

```diff
@@ -67,11 +67,20 @@ public UIParent(object parent, bool useEmbeddedWebview)
             ThrowPlatformNotSupported();
         }
 
+        /// <summary>
+        /// Checks if the system weview can be used. 
+        /// </summary>
+        public static bool IsSystemWebviewAvailable() // This is part of the NetStandard "interface" 
+        {
+            ThrowPlatformNotSupported();
```

Comment:
If this is a capability check, why should this throw?  This should just return false.